tss/rsa/util.go

```diff
@@ -0,0 +1,26 @@
+package rsa
+
+import (
+	"crypto/rsa"
+	"math/big"
+)
+
+var ONE = big.NewInt(1)
+
+func calculateDelta(l int64) *big.Int {
```

Comment:
```suggestion
func calculateDelta(l uint64) *big.Int {
```

Reply:
calculateDelta calls MulRange which requires an int64. This is an internal function so there's no reason to force the conversion if it's not needed.